BigQuery/src/BigQueryClient.php

```diff
@@ -455,6 +455,8 @@ public function job($id, array $options = [])
      *           before or at this timestamp are returned.
      *     @type int $minCreationTime Milliseconds since the POSIX epoch. If set, only jobs created
      *           after or at this timestamp are returned.
+     *     @type string $parentJobId If set, show only child jobs of the
```

Comment:
this PR just adds the doc string, the impl is just autogenerated?

Reply:
Correct. The implementation works based on the discovery doc. This PR documents the field and makes sure it works as expected with the test.